tests/nfa/nfa.cc

```diff
@@ -2982,3 +2982,28 @@ TEST_CASE("Mata::Nfa:: create simple automata") {
     CHECK(is_in_lang(nfa, { { 0, 1, 2, 2, 0, 1, 2, 1, 0, 0, 2, 1 }, {} }));
     CHECK(!is_in_lang(nfa, { { 3 }, {} }));
 }
+
+TEST_CASE("Mata::Nfa:: print_to_mata") {
```

Comment:
For harder test,  you could traverse all files in https://github.com/VeriFIT/mata/tree/devel/tests-performance/automata directory and do parse/print and comparison as well.

Reply:
We are confident that the introduced changes work as expected, and we do not need to test it further. If we ever find any bugs with the print function, we will rethink our stance on this issue. We want this merged as quickly as possible so that we can start using it for the currently worked-on features.